expo-maps/android/src/main/java/expo/modules/maps/PropsData.kt

```diff
@@ -1,8 +1,21 @@
 package expo.modules.maps
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
 
 enum class MapType(val value: String) {
     Normal("normal"),
     Hybrid("hybrid"),
     Satellite("satellite"),
     Terrain("terrain")
 }
+
+class MarkerObject : Record {
+    @Field
+    val type: String? = null
+
+    @Field
+    val lat: Double? = null
+
+    @Field
+    val lng: Double? = null
```

Comment:
It seems like, I can't just do 
```suggestion
    @Field
    val lng: Double? 
```
![Zrzut ekranu 2022-01-18 o 00 34 14](https://user-images.githubusercontent.com/55145344/149847682-173e0faa-b2f7-4285-b652-b83f539c965a.png)

@lukmccall Letting you know just in case if this is bug, because it looks like it should be possible in [docs](https://docs.expo.dev/modules/module-api/#records).

Reply:
It is a bug in the documentation. In Kotlin, you can't create a "normal" property without a value. In that case, I would do one of two things:
- `val lng: Double = 0`
- `lateinit var lng: Double` 

The first one will provide a default value if `lng` wasn't provided by the user.